frontend/src/pages/auth/sign-in.tsx

```diff
@@ -0,0 +1,154 @@
+import { useState } from "react";
+import { AuthButton, AuthContainer, AuthContent, AuthFooter, AuthForm, AuthHeader, AuthInput, AuthLabel, AuthSub, AuthSwitch, AuthTitle } from "./auth.styles";
+import axios from "axios";
+import { toast } from "react-toastify";
+import { useGlobalContext } from "../../contexts/global.context";
+import { useNavigate } from "react-router-dom";
+import { handleAPIError } from "../../utils/errors.util";
+
+interface SignInInterface {
+    brand_email: string,
+    brand_password: string,
+    [key: string]: string
+}
+
+interface AuthInputInterface {
+    name: string,
+    label: string,
+    type: string,
+    placeholder: string
+}
+
+export default function SignIn() {
+
+    const { handleLogIn } = useGlobalContext()
+
+    const navigate = useNavigate()
+
+    const [authForm, setAuthForm] = useState<SignInInterface>({
+        brand_email: '',
+        brand_password: ''
+    })
+
```

Comment:
We might need a `useEffect` to pull away the user from the sign-in/sign-up pages, if they're already logged in. Why is this not implemented?